Pipfile

```
@@ -5,24 +5,25 @@ verify_ssl = true
 
 [dev-packages]
 pipenv-setup = {editable = true,path = "."}
-pytest = {markers = "python_version>='3.6'",version = "~=5.2"}
-pytest-mypy = {markers = "python_version>='3.5'",version = "~=0.3"}
-pytest-cov = "~=2.7"
-pytest-datadir = "~=1.3"
-pytest-xdist = "~=1.29"
-tox = "~=3.14"
-autopep8 = "~=1.4"
+pytest = "*"
+pytest-mypy = "*"
+pytest-cov = "*"
+pytest-datadir = "*"
+pytest-xdist = "*"
+tox = "*"
+autopep8 = "*"
 
 [packages]
-pipfile = "~=0.0"
-black = {markers = "python_version>='3.6'",version = "==19.10b0"}
-colorama = "~=0.4"
-packaging = "~=20.0"
-requirementslib = "~=1.5"
-vistir = "~=0.4"
-autopep8 = {markers = "python_version<'3.6'",version = "~=1.4"}
-six = "~=1.12"
-typing = {markers = "python_version<'3.7'", version = "~=3.7"}
+pipfile = "*"
+black = {markers = "python_version>='3.6'",version = "==21.9b0"}
```

Why not leave `black` unpinned here? 
```suggestion
black = {markers = "python_version>='3.6'", version="*"}
```

Pinning It in the `dev` extras makes more sense to me

---

One thing I've noticed with black, with is quite frustrating, is that it cannot be unpinned without causing other problems. Pipenv will refuse to install it as a "pre-release" without adding the additional flag. This long traceback can be a bit off-putting for some users. Though easily resolved it could also break any scripts that use Pipenv and do not include this flag.

---

Yup I was able to confirm.

Additional information:

- issue pypa/pipenv#1760 has been open for a few years now 
- PR pypa/pipenv#4607 was opened early this year, but I'm unsure of its status

I'm interested if it's feasible to remove `black` as a minimal dependency. It seems like we could provide it as an `extra` in one way or another